File: Feitos em C/2540.c
#include <stdio.h>
int main() {
	int times,i;
    while (scanf("%d",&times)==1) {
        int imput [times];
        int impeachment = 0;
        for(i = 0; i < times; i++) {
            scanf("%d",&imput[i]);
            if (imput[i] == 1) {
                impeachment++;
            }
        }
        double n = (times * 2) / 3.0;
        if (impeachment >=  n) {
            printf("impeachment\n");
        }
        else {
        	printf("acusacao arquivada\n");
        }
    }
    return 0;
}

File: Feitos em Java/2540.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        while (leitor.hasNext()) {
            int times = leitor.nextInt();
            int[] imput = new int [times];
            int impeachment = 0;
            for(int i = 0; i < times; i++) {
                imput[i]=leitor.nextInt();
                if (imput[i] == 1) {
                    impeachment++;
                }
            }
            double n = (double) (times * 2) / 3;
            if (impeachment >=  n) {
                System.out.println("impeachment");
            }
            else {
                System.out.println("acusacao arquivada");
            }
        }
    }
}